marketplace_builder/views/pages/tutorials/data-import-export/import.liquid

```diff
@@ -0,0 +1,104 @@
+---
+converter: markdown
+metadata:
+  title: Data Import
+  description: Import data
+slug: tutorials/data-import-export/import
+searchable: true
+---
+For now you can export/import instance data on new stack. Data to be imported needs to be in json format:
+
+```json
+{
+  "users": [...],
+  "models": [...],
+  "transactables": [...]
+}
+```
+
+You can generate data from existing instance with `marketplace-kit data export` command([more details here](/tutorials/data-import-export/export#cli-export)). It will generate `data.json` file. You can also import data from other sources(old stack, other systems) once you generate proper JSON file. This is fairly easily to convert for example csv file into json file.
+
+## General information
+
+Before import, your model schemas need to be deployed.
+
+When importing data `id` fields will be chanaged. Relations will be keept. If you have your own relations defined in properties you need to first decribe them in model schema files - [more info here](/tutorials/data-import-export/models-relations).
+
+Importing always adds data to the instance. When you run import twice it will try add data to existing set. If you test you imports and need to clean instance data use `marketplace-kit data clean`. Note that this command will work only on staging instanes.
```

Comment:
```suggestion
Importing always adds data to the instance. When you run import twice it will try to add data to existing set. If you test your imports and need to clean instance data use `marketplace-kit data clean`. Note that this command will work only on staging instances.
```